about/release-notes.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Timescale release notes and future plans
```

Comment:
we don't include any future plans, not sure why we say this.

Also, these are Timescale release notes for self-hosted products.

Reply:
Removed the future plans comment. I'd prefer not to specify self-hosted as we use all these products in Timescale Cloud.